src/views/MediaChannels/messages.ts

```diff
@@ -0,0 +1,7 @@
+import { defineMessages } from "react-intl";
```

Comment:
How can we empower translators to manage strings without requiring developer intervention?

Reply:
I remember something about lookalise in JMP?

Reply:
There are tools for that, but that need some investigation.